src/server/controllers/users.controller.js

```diff
@@ -39,7 +39,7 @@ const FB_GRAPH_CRAWL_PARAMS = 'name,email,locale,timezone,education,work,locatio
 
 // Return my profile.
 export function getMyProfile(req, res, next) {
-  User.findOne({ _id: req.user._id }).exec()
+  User.findById(req.user._id)
```

Comment:
It seems pretty good.
Are you gonna change rest of the functions that access to database?